core/src/components/modal/gestures/sheet.ts

```diff
@@ -143,8 +162,35 @@ export const createSheetGesture = (
      * compute where the offset of the animation should be
      * relative to where the user dragged.
      */
-    const initialStep = 1 - currentBreakpoint;
-    offset = clamp(0.0001, initialStep + (detail.deltaY / height), 0.9999);
+    const initialBreakpoint = 1 - currentBreakpoint;
+    const secondToLastBreakpoint = 1 - breakpoints[1];
+    const step = initialBreakpoint + (detail.deltaY / height);
+    const isAttempingDismissWithCanDismiss = step >= secondToLastBreakpoint && canDismissBlocksGesture;
```

Comment:
~ I tried using a sheet modal with only one breakpoint (0.5) to see if that breaks this due to accessing `breakpoints[1]` directly, but it seemed to work without any issues? I'm not really sure why, but I figured I'd mention that in case you have an "oh shoot" moment, haha.

Reply:
Good catch, I should probably add a safe guard for it. 

I think why it doesn't break is:

`const secondToLastBreakpoint = 1 - breakpoints[1]` which equals `NaN`

Then doing `0.5 >= NaN` will return `false`, so `isAttempingDismissWithCanDismiss` is never `true`.